west.yml

```diff
@@ -199,7 +199,7 @@ manifest:
       groups:
```

Comment:
PR https://github.com/zephyrproject-rtos/zephyr/pull/481 points to Zephyr change. Did you mean PR from hal_nxp?

Reply:
ah, yes, it's supposed to refer to https://github.com/zephyrproject-rtos/hal_nxp/pull/481. The commit description is only temporary and will update it once HAL changes are merged.